src/main/java/org/sonatype/nexus/plugins/conda/internal/CondaRecipeSupport.groovy

```diff
@@ -115,35 +115,35 @@ abstract class CondaRecipeSupport
   //public static final String filenameIndexHtml...
 
   static Matcher rootChannelIndexHtmlMatcher() {
-    buildTokenMatcherForPatternAndAssetKind('/index.html', AssetKind.CHANNEL_INDEX_HTML, GET, HEAD)
+    buildTokenMatcherForPatternAndAssetKind('/{path:.+}/index.html', AssetKind.CHANNEL_INDEX_HTML, GET, HEAD)
```

Comment:
could/should these reference the constants in CondaPathUtils.java, e.g.: CondaPathUtils.INDEX_HTML

Reply:
Yeah, I can either do that in this PR or separately. No care either way!